db/migrate/20160127140802_add_digital_location_to_delivery_location.rb

```diff
@@ -0,0 +1,5 @@
+class AddDigitalLocationToDeliveryLocation < ActiveRecord::Migration
+  def change
+    add_column :locations_delivery_locations, :digital_location, :boolean
+  end
```

Comment:
Setting a default seems like a good idea, to minimize the manual editing required -- should be it be true?